src/zkp/pilog.rs

```diff
@@ -43,7 +42,7 @@ pub(crate) struct PiLogProof {
 pub(crate) struct PiLogInput {
     setup_params: ZkSetupParameters,
     q: BigNumber,
-    N0: BigNumber,
+    pk: PaillierEncryptionKey,
```

Comment:
nit: Add documentation here to note that this correlates to `N0` in the text.

Reply:
I added a comment, but did you mean as external documentation?

Reply:
I meant field documentation, yeah. `///`.